2 - RTB/Documentazione interna/Verbali/23-12-31/23-12-31.typ

```diff
@@ -0,0 +1,68 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "31/12/23",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Banzato Alessio",
+  ),
+  missingMembers: (
+    "Gardin Giovanni",
+  ),
+  timeStart: "15:00",
+  timeEnd: "16:00",
+);
+
+= Ordine del giorno
+- Valutazione dell'avanzamento;
+- Analisi di retrospettiva;
+- Pianificazione.
+
+== Valutazione dell'avanzamento
+Ogni membro del gruppo ha esposto ciò che ha svolto durante lo sprint rispetto al prodotto software o documentale che aveva in carico.
+
+=== Piano di Qualifica
+- Sono state individuate delle metriche di base riguardanti i documenti e l'andamento generale del team;
+- È stato creato un cruscotto di qualità su Grafana per monitorare l'andamento del gruppo tramite misurazione degli story points, i quali vengono classificati in quattro stati:
+  + in corso;
+  + in verifica;
+  + completato;
+  + da completare.
+
+=== Norme di Progetto
+Non sono stati prodotti tutti i capitoli preventivati anche a causa di un prevedibile rallentamento dei lavori discusso nella @improvements.
+
+=== Analisi dei Requisiti
+È stato implementato il tracciamento requisito-fonte. È necessaria una revisione generale del documento.
+
+=== Piano di Progetto
+Non sono state rilevate criticità, la stesura del documento procede senza problemi.
+
+=== PoC
+È stato realizzato il PoC finale. Dato il cambio di ruoli è stato programmato un meeting interno per spiegare nel dettaglio il PoC e il suo funzionamento ai nuovi Programmatori. Sarà inoltre necessario realizzare un video demo da mostrare al Proponente.
```

Comment:
Più che "E' stato realizzato il PoC finale", metterei in luce che è frutto della compattazione dei "mini PoC" finora prodotti (e che hanno trovato approvazione del Proponente). Non perchè sia sbagliato quanto detto, ma lo preciserei maggiormente ecco.